cypress.config.js

```diff
@@ -0,0 +1,15 @@
+const { defineConfig } = require('cypress')
+
+module.exports = defineConfig({
+  env: {
+    API_URL: 'http://localhost:3001',
+    hideCredentials: true,
+    requestMode: true
+  },
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    experimentalRunAllSpecs: true // permite rodar todos os testes visuais de uma vez só
```

Comment:
```suggestion
    experimentalRunAllSpecs: true // permite rodar todos os testes de uma vez só
```

Tal configuração permite rodar todos os testes, independente se são visuais ou não.

Comentários devem ser o mais claro possíveis para evitar qualquer confusão.

Reply:
Obrigada pela dica!